docs/_index.yaml

```diff
@@ -27,26 +27,19 @@ landing_page:
         classname: button
   - classname: devsite-landing-row-cards
     items:
-    - heading: "The What-If Tool: Code-Free Probing of Machine Learning Models"
+    - heading: "TensorBoard.dev: a new way to share your ML experiment results"
       image_path: /resources/images/tf-logo-card-16x9.png
-      path: https://ai.googleblog.com/2018/09/the-what-if-tool-code-free-probing-of.html
-      buttons:
-      - label: "Read on Google AI blog"
-        path: https://ai.googleblog.com/2018/09/the-what-if-tool-code-free-probing-of.html
-    - heading: "Interactive supervision with TensorBoard"
-      image_path: /resources/images/tf-logo-card-16x9.png
-      path: https://medium.com/tensorflow/interactive-supervision-with-tensorboard-9a101c91d3f7
+      path: https://blog.tensorflow.org/2019/12/introducing-tensorboarddev-new-way-to.html
       buttons:
       - label: "Read on TensorFlow blog"
-        path: https://medium.com/tensorflow/interactive-supervision-with-tensorboard-9a101c91d3f7
+        path: https://blog.tensorflow.org/2019/12/introducing-tensorboarddev-new-way-to.html
+    - heading: "What's new in TensorBoard (TF Dev Summit '19)"
+      youtube_id: xM8sO33x_OU
+      buttons:
+      - label: Watch the video
+        path: https://www.youtube.com/watch?v=xM8sO33x_OU
     - heading: "Hands-on TensorBoard"
```

Comment:
Is it worth adding "(TF Dev Summit '17)" for clarity? The code is likely slightly out of date in the tutorial